### ISSUE TITLE
Plans for grouping projects? "paths" / "group" or another?

### ISSUE THREAD

**Issue by Taeluf** [alefragnani/vscode-project-manager#415]
I would think I could set the same "group" for three different projects and have them group together in the project viewer.

That or set multiple paths for one project & be able to select which path to open.

It would also be cool to open a project rootPath as a sub-selector. Aka, set rootPath to `~/dev/PHP`, then when I click the project, I'd get to choose from any sub-directory of `~/dev/PHP`

I'm thinking about contributing, but I want to know what the intent is/discuss it so (if I do make the time to contribute) I work within the plans for this project.

P.S. Love this extension <3

**Comment by alefragnani** [comment]
Hi @Taeluf ,

First of all, thank you. I'm glad to know you like the extension.

For grouping projects, there is already #50 reported. It's one of my next priorities, once I recently added support to _remote development_. It will not be strictly "grouping", but you will have a similar behavior.

> That or set multiple paths for one project & be able to select which path to open.

A **project** is a folder or a workspace (multi-root), but not a _shortcut_ for choose projects. I guess your best bet is to use use workspaces and have all folders opened at once on the same window.

About contributing, some pieces of the extension are not public right now so, you won't be able to clone/build. I'm planning to make it public in the near future.

Hope this helps

**Comment by Taeluf** [comment]
Hi :) 
I pinned that issue to look at later. Thanks for your quick reply & sorry I overlooked the existing issue.

That would be cool to be able to contribute, even if the source code license is restrictive. 

My hacky solution is to have a dir at `~/dev/php`, then create a symlink at `~/dev/php/php` that points to `~/dev/php`. Then I open `~/dev/php/php` via project manager & immediate ctrl+k ctrl+o (which always opens the parent directory). It opens me to `~/dev/php` & I can select the PHP project I wanna work on. 

I wasn't aware of workspaces, but I checked them out a little bit. Thanks for the suggestion!
